src/main/java/ch/njol/skript/lang/Expression.java

```diff
@@ -21,11 +21,13 @@
 
 import java.util.Arrays;
 import java.util.Iterator;
+import java.util.stream.Stream;
 
 import org.bukkit.event.Event;
 import org.bukkit.inventory.ItemStack;
 import org.eclipse.jdt.annotation.Nullable;
 
+import com.google.common.collect.Streams;
```

Comment:
We don't want to rely on Guava for core Skript functionality. StreamSupport API of Java should be good enough in this case.